paddle/framework/error.h

```diff
@@ -122,15 +117,23 @@ class Error {
    */
   bool isOK() const { return msg_ == nullptr; }
```

Comment:
I am not very familiar with C++ naming convention so I could be wrong: to me it seems `ok()` is better than `isOK()`, "is" makes develop type two more characters, and does not provide much value in my opinion.